polar-core/src/types.rs

```diff
@@ -114,9 +114,10 @@ impl Symbol {
 }
 
 #[derive(Debug, Clone, Serialize, Deserialize, Eq, PartialEq, Hash)]
-pub struct Predicate {
+pub struct Call {
     pub name: Symbol,
     pub args: TermList,
+    pub kwargs: Option<BTreeMap<Symbol, Term>>,
```

Comment:
Is this a BTreeMap b/c of the derivation of Hash, or is there another reason? Just curious b/c I wouldn't think the order of the kwargs would matter (though maybe it does for printing purposes?

Reply:
Yeah, just did it because that's what `Fields` is (for instance literals).